src/modules/timepix3/handlers/TimingEventHandler.cpp

```diff
@@ -8,6 +8,8 @@
 //===----------------------------------------------------------------------===//
 
 #include <common/debug/Trace.h>
+#include <netinet/in.h>
+#include <cstdint>
 #include <handlers/TimingEventHandler.h>
```

Comment:
Suggested:
#include <timepix3/handlers/TimingEventHandler.h>

The other one obviously also works but it makes it a bit clearer where this belongs